### ISSUE TITLE
Test other Apify methods compatibility

### ISSUE THREAD

**Issue by JJetmar** [JJetmar/apify-sdk-legacy#1]
Test
```
Apify.events
Apify.main
Apify.openKeyValueStore
Apify.getInput
Apify.setValue
Apify.PuppeteerCrawler
Apify.Session
Apify.pushData
Apify.isAtHome
```

**Comment by JJetmar** [comment]
Also `Apify.newClient` and `Apify.getEnv`.

**Comment by JJetmar** [comment]
Currently implemented:
```
Apify.events
Apify.openKeyValueStore
Apify.getInput
Apify.setValue
Apify.PuppeteerCrawler
Apify.pushData
Apify.isAtHome
```
Missing: 
```
Apify.main
Apify.Session
Apify.newClient
```

**Comment by JJetmar** [comment]
Fixed in version `1.0.5`
